lib/seed/production_experiment_test_seeder.rb

```diff
@@ -0,0 +1,24 @@
+module Seed
+  class ProductionExperimentTestSeeder
+    TEMPORARY_PHONE_NUMBER = "8675309"
```

Comment:
```suggestion
    TEMPORARY_PHONE_NUMBER = "0001234"
```

Might as well make sure this is really an invalid number, just in case.

Reply:
Done.